GitUp/Application/AppDelegate.m

```diff
@@ -277,6 +281,16 @@ - (void)_updatePreferencePanel {
   }
 }
 
+- (void)_updateThemePopUpButton {
+  NSString* theme = [[NSUserDefaults standardUserDefaults] stringForKey:kUserDefaultsKey_Theme];
+  for (NSMenuItem* item in _themePopUpButton.menu.itemArray) {
+    if ([item.title isEqualToString:theme]) {
```

Comment:
I know GitUp isn't localized in other languages, but generally speaking it's bad practice to compare user facing strings to the keys backing the data (because another language likely won't have the same word). Let's make sure we're comparing either the representedObject (see `_channelPopUpButton` menu items as an example) or you could use tag number.

```
[_channelPopUpButton.menu removeAllItems];
for (NSString* string in @[ kReleaseChannel_Stable, kReleaseChannel_Continuous ]) {
  NSMenuItem* item = [[NSMenuItem alloc] initWithTitle:NSLocalizedString(string, nil) action:NULL keyEquivalent:@""];
  item.representedObject = string;
  [_channelPopUpButton.menu addItem:item];
}
```

Reply:
Yes I was trying to use `representedObject` initially but it didn't work very well with storyboard. Will fix it later this week and clean up the dirty code.